client/components/LoginPage.tsx

```diff
@@ -1,44 +1,93 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { 
+  TextField, 
+  Box, 
+  Typography, 
+  InputAdornment,
+  IconButton,
+  Button
+} from "@mui/material";
```

Comment:
This imports the entire mui component library, when you only need a few specific components. Depending on dev setup, this can be very very bad for bundle sizes (I'm not 100% sure if Vite handles this through what is called "tree shaking" but I have seen some evidence from people that this has caused issues for them with Vite builds, so I'm suspecting it will not protect from this). This is why the MUI documentation examples mostly do not import components this way. I recommend instead importing each component directly from its module

```suggestion
import TextField from "@mui/material/TextField";
import Box from "@mui/material/Box";
import Typography from "@mui/material/Typography";
import InputAdornment from "@mui/material/InputAdornment";
import IconButton from "@mui/material/IconButton";
import Button from "@mui/material/Button";
```

Reply:
Got it. I actually did not know the difference. Definitely don't want to import more than is needed